src/stylesheets/index.css

```diff
@@ -0,0 +1,37 @@
+body {
+  font: 14px "Century Gothic", Futura, sans-serif;
+  margin: 20px;
+}
+
+ol, ul {
+  padding-left: 30px;
+}
+
```

Comment:
Separate the CSS by component or page please, not one large file

Reply:
At the moment, I only have one page. I turned what were separate pages into functions in Index.